webapp/src/components/admin_console_settings/rtcd_service_url/index.tsx

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {ChangeEvent, useState} from 'react';
+import {CustomComponentProps} from 'src/types/mattermost-webapp';
+import {useDispatch, useSelector} from 'react-redux';
+
+import styled from 'styled-components';
+
+import {FormattedMessage} from 'react-intl';
+
+import {adminStats, isCloud, isOnPremNotEnterprise} from 'src/selectors';
+import {PrimaryButton} from 'src/components/buttons';
+import {HorizontalSpacer, VerticalSpacer} from 'src/components/shared';
+import {modals} from 'src/webapp_globals';
+import {
+    IDOnPremTrialError,
+    IDOnPremTrialSuccess,
+    OnPremTrialError,
+    OnPremTrialSuccess,
+} from 'src/components/admin_console_settings/rtcd_service_url/modals';
+import {requestOnPremTrialLicense} from 'src/actions';
+
+const RTCDServiceUrl = (props: CustomComponentProps) => {
+    const dispatch = useDispatch();
+    const restricted = useSelector(isOnPremNotEnterprise);
+    const cloud = useSelector(isCloud);
+    const stats = useSelector(adminStats);
+
+    const [currentRestricted, setCurrentRestricted] = useState(restricted);
+
+    const leftCol = 'col-sm-4';
+    const rightCol = 'col-sm-8';
+
+    // Webapp doesn't pass the placeholder setting.
+    const placeholder = 'https://rtcd.example.com';
```

Comment:
I think we can import this from `src/manifest.` Also, would it make sense to update `plugin.json` and set the type of setting as `custom`?

Reply:
Good idea! Done.